src/explore-education-statistics-common/src/components/form/FormFieldset.tsx

```diff
@@ -59,7 +59,7 @@ const FormFieldset = ({
             'govuk-fieldset__legend',
             `govuk-fieldset__legend--${legendSize}`,
             {
-              'govuk-visually-hidden': legendHidden,
+              'govuk-visually-hidden govuk-!-font-size-14': legendHidden,
```

Comment:
I'm a little confused at how this works. How is the visually hidden legend actually causing overflow issues? And why is the solution to make the font size smaller?

Seems to me like there's a more foundational issue here that requires a different styling fix than changing font size. Like what if the legend is just so long is still causes this overflow issue even with the smaller font?